packages/apps/human-app/server/src/integrations/exchange-oracle-api/exchange-oracle-api.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { HttpModule } from '@nestjs/axios';
+import { ExchangeOracleApiGateway } from './exchange-oracle-api.gateway';
+import { ExchangeOralceApiProfile } from './exchange-oracle-api.mapper';
+
+@Module({
+  imports: [HttpModule],
+  providers: [ExchangeOracleApiGateway, ExchangeOralceApiProfile],
+  exports: [ExchangeOracleApiGateway],
+})
+export class ExchangeOracleApiModule {}
```

Comment:
packages/apps/human-app/server/src/integrations/reputation-oracle/reputation-oracle.module.ts follows a different naming convention, it does not contain the Api part, please align